index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  <title>Crypto New World</title>
+</head>
+<body>
+  <main class="container">
+    <header>
+      <div class="mobile-header">
+        <nav class="nav-bar-mobile">
```

Comment:
- [ ] Please put the `<header>` out of the `<main>.` It's not a good practice to place `<header>` inside the `<main>` because they are intended to be at the same level of hierarchy. The main tag should only be responsible for the main /hero section of the webpage.